cypress.json

```diff
@@ -2,9 +2,12 @@
   "baseUrl": "http://localhost:57021",
   "integrationFolder": "./src",
   "testFiles": "**/*.test.*",
+  "ignoreTestFiles": [
+    "**/tests/unit/*.test.*"
+  ],
```

Comment:
Consider changing our `testFiles` to `./src/**/*.test.*` instead of using `ignoreTestFiles`.

[`ignoreTestFiles` seems to be legacy configuration](https://docs.cypress.io/guides/references/legacy-configuration#Folders--Files). It works because we are currently using an older version of Cypress but not when we upgrade.

Reply:
I changed the globbing pattern :)